pkg/controller/propagator/propagation.go

```diff
@@ -46,7 +46,9 @@ var templateCfg templates.Config
 func Initialize(kubeconfig *rest.Config, kubeclient *kubernetes.Interface) {
 	kubeConfig = kubeconfig
 	kubeClient = kubeclient
-	templateCfg = templates.Config{StartDelim: "{{hub", StopDelim: "hub}}"}
+	// Adding four spaces to the indentation makes the usage of `indent N` be from the logical
+	// starting point of the resource object wrapped in the ConfigurationPolicy.
+	templateCfg = templates.Config{AdditionalIndentation: 4, StartDelim: "{{hub", StopDelim: "hub}}"}
```

Comment:
i dont think its 4,  its 4 indent "units" ... did you test ?

Reply:
@ckandag my mistake. 16 is the number that worked for me locally.